app/views/shared/_improve_this_page.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="improve-this-page" data-module="improve-this-page">
+  <div class="js-prompt">
+    <h3 class="improve-this-page__is-useful-question">Is this page useful?</h3>
+    <%= link_to contact_govuk_path, { class: 'button js-page-is-useful', role: 'button' } do %>
+      Yes <span class="visually-hidden"> this page is useful</span>
+    <% end %>
+    <%= link_to contact_govuk_path, { class: 'button js-offer-feedback', role: 'button' } do %>
+      No <span class="visually-hidden"> this page is not useful</span>
+    <% end %>
+    <%= link_to contact_govuk_path, { class: 'button js-offer-feedback improve-this-page__anything-wrong', role: 'button' } do %>
+      Is there anything wrong with this page? <span class="visually-hidden"> this page is not useful</span>
+    <% end %>
+  </div>
+  <div class="js-feedback-form js-hidden">
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <div class="js-errors"></div>
+        <form>
+          <input type="hidden" name="path" value="<%= request.original_url -%>">
+          <input type="hidden" name="user_agent" value="<%= request.user_agent -%>">
+          <div class="improve-this-page__section">
+            <label for="improve-this-page_description">How should we improve this page? <span class="visually-hidden">required</span></label>
+            <textarea name="description" rows="5" required id="improve-this-page_description"></textarea>
```

Comment:
To avoid confusion between BEM classes and IDs, is it worth just using a hyphen here (`improve-this-page-description`)? It can be a bit hard to parse the classes and IDs in this block as they're so similar.

(Alternative suggestion – can the association between label and input be made implicit by wrapping the input with the label? Then the `for` and `id` attributes can be removed)